server/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Base image
-FROM python:3.6
+FROM registry.access.redhat.com/ubi8/python-36:latest
 
-# Update installation utilites and packages
-RUN apt-get update
+USER root
```

Comment:
I made a ticket to remove this

Reply:
[COCO-85](https://freshworks.atlassian.net/browse/COCO-85)